cpg_workflows/query_modules/seqr_loader_cnv.py

```diff
@@ -185,6 +165,88 @@ def annotate_cohort_gcnv(
         transcriptConsequenceTerms=gene_major_consequences.extend(default_consequences),
         docId=mt.variantId[0:512],
     )
+    # drop the variant_qc. Can't trust it NAN values
+    mt = mt.drop('variant_qc')
 
     # write this output
     mt.write(out_mt_path, overwrite=True)
+
+
+def annotate_dataset_sv(mt_path: str, out_mt_path: str):
+    """
+    process data specific to samples in this dataset
+    do this after sub-setting to specific samples
+
+    Args:
+        mt_path (str): path to the annotated MatrixTable
+        out_mt_path (str): and where do you want it to end up?
+    """
+
+    logging.info('Annotating genotypes')
+
+    mt = read_hail(mt_path)
+
+    mt = mt.annotate_rows(
+        genotypes=hl.agg.collect(
+            hl.struct(
+                sample_id=mt.s,
+                gq=mt.CNQ,
+                cn=mt.CN,
+                end=mt.end,
+                num_exon=hl.missing('int32'),
```

Comment:
Can we get num_exon from anywhere? It is one of the most useful values for filtering so it would be great to have it.